Sources/VergeStore/StateReader.swift

```diff
@@ -123,52 +163,111 @@ extension StateReader {
   ///   - store:
   ///   - content:
   public init<Store: StoreType>(
-    _ store: Store,
-    @ViewBuilder content: @escaping (Changes<Value>) -> Content
+    _ store: Store
   ) where Value == Store.State, Value : Equatable {
-    self.init(store, .map(\.root), content: content)
+    self.init(store.derived(.map(\.root)))
   }
 
-  /// Initialize from `Derived`
+  /// Initialize from `Store`
   ///
-  /// - Complexity: Depends on the map parameter
+  /// - Complexity: ✅ Using implicit drop-input with Equatable
   /// - Parameters:
-  ///   - derived:
+  ///   - store:
   ///   - content:
   public init<Derived: DerivedType>(
-    _ derived: Derived,
-    _ map: MemoizeMap<Changes<Derived.Value>, Value>,
```

Comment:
Passing MemoizeMap causes the complexity of auto-completion and verbosity of signature.
In most of the cases, it's enough passing Derived object.

If we need to create chained Derived, pass the chained Derived from outside.